### PR TITLE
Adds AEG back to protolathe

### PR DESCRIPTION
**Pull request by icantthinkofanameritenow**
Long story short, between Vaults being a thing and a complete lack of cooperation between Security and Science I'm adding AEGs back to the protolathe again. They're Nanotrasen experimental technology now, so you'll need to ask Cargo for the NT Experimental crate first.

### CODE REVIEW

**Comment by Shadowmech88** [comment]
Why not leave the modkit in so you can give them to Security members that come asking instead of needing to get the crate first?
Have both options.

**Comment by icantthinkofanameritenow** [comment]
@Shadowmech88 
Good idea. I think I'll add a single AEG modkit to each of the maps' armories as well, so no matter what happens there's always going to be at least one available.

**Comment by ghost** [comment]
Is the cost of the upgrade going to be reduced? If I recall correctly it is equal to the cost of an AEG.

**Comment by icantthinkofanameritenow** [comment]
@DrCelt
The modkit simply doesn't show up in the protolathe any longer- why would it when you can just make the AEG without using an energy gun first? Instead, I'm going to add one modkit to the Secure Armory on each map directly in either this or a future PR.

File: code/modules/research/designs/weapons.dm [hunk 1]
```diff
@@ -148,7 +147,6 @@
 	reliability_base = 100
 	category = "Weapons"
 	build_path = /obj/item/weapon/reagent_containers/spray/chemsprayer
-	req_lock_access = list(access_medical, access_cmo)
```
Comment (ghost) [inline code/modules/research/designs/weapons.dm]: Atomic PR needed for this.

**Comment by aceedex** [comment]
I like this, this is nice.

Now just to see comdoms rushing the armory roundstart for their recharging stungun

**Comment by Shadowmech88** [comment]
I was actually suggesting leaving the modkit in the protolathe so that _if_ a security officer comes asking for an AEG, they can just get the kit instead of having to wait for the scientist to go to Cargo.

Is there any downside to leaving the modkit in the protolathe?

File: code/modules/research/designs/weapons.dm (hunk 1 above)
Comment (icantthinkofanameritenow) [inline code/modules/research/designs/weapons.dm]: That line doesn't actually seem to work ingame anyway. Should I just revert it?

**Comment by icantthinkofanameritenow** [comment]
@Shadowmech88
Not really, I just thought there would be no point, since the officer would still need to get he egun somehow and that would require either the armory or Cargo.

File: code/modules/research/designs/weapons.dm (hunk 1 above)
Comment (ComicIronic) [inline code/modules/research/designs/weapons.dm]: Just don't touch code that's not relevant to your PR, regardless of what you're doing with it.

Comment (ghost) [inline code/modules/research/designs/weapons.dm]: Yeah, it doesn't spawn a lockbox, that's an oversight.

**Comment by 9600bauds** [comment]
Why exactly remove the modkit from the Protolathe

**Comment by icantthinkofanameritenow** [comment]
@9600bauds 
They aren't being removed now. I was considering it earlier due to me thinking nobody would make the modkits when they could just make whole AEGs from scratch, but after some thought I realized I needed a contingency in case Cargo staff were absent or too busy doing their own shit  to order the crate.

**Comment by Probe1** [comment]
nah its better this way

people cooperate and the AEG isn't the only gun that is printed anymore.

**Comment by Probe1** [comment]
Also I'm gonna say it because I heard you bitching about people bitching.

You make balance PRs without asking anyone what they think then get confused when people tell you to shove it.

**Comment by Intigracy** [comment]
You misspelled modkit in the changelog.

File: code/modules/research/designs/nanotrasen.dm [hunk 2]
```diff
@@ -31,3 +31,13 @@
 	build_path = /obj/item/weapon/gun/gravitywell
 	locked = 1
 	req_lock_access = list(access_rd)
+
+/datum/design/nuclear_gun
```
Comment (Intigracy) [inline code/modules/research/designs/nanotrasen.dm:35]: I'm almost positive that giving these two designs the exact same path is going to cause a problem. Change the modkit's to nuclear_gun_modkit or something

**Comment by clusterfack** [comment]
This is a shit idea, you should already be able to make AEGs en masse if cargo and research work together. This brings back the old shitty problems except this time you just need a crate to unlock them.

A step fucking backwards.

**Comment by icantthinkofanameritenow** [comment]
While I'm leaving this branch for the time being, I'm starting to think I'm going about this wrong. Instead of making AEGs accessible, we should be forcing RnD to interact more. No amount of coding by me or anyone else can fix that.

So for the moment, if you need AEGs badly enough to whine about it, you have enough energy to move your ass over to Cargo and order an energy weapons crate.

File: code/modules/research/designs/nanotrasen.dm (hunk 2 above)
Comment (PJB3005) [inline code/modules/research/designs/nanotrasen.dm:35]: It does cause problems.

**Comment by icantthinkofanameritenow** [comment]
Changed my mind about the Phazon thing- given the choice between this needing phazon and ordering a crate, I have to hold out some kind of hope that Cargo can keep its shit together enough so if the station does need to be armed ASAP they can take the pains to ensure it can be armed.

**Comment by ComicIronic** [comment]
Rejected.

**Pull request closed without merging**